config/grafana/overview.json

```diff
@@ -715,9 +709,9 @@
             "uid": "P1809F7CD0C75ACF3"
           },
           "exemplar": true,
-          "expr": "rate(controller_runtime_reconcile_time_seconds_sum{controller=\"manifest\"}[$__rate_interval])\n/\nrate(controller_runtime_reconcile_time_seconds_count{controller=\"manifest\"}[$__rate_interval])",
+          "expr": "rate(reconcile_duration_seconds_sum[$__rate_interval])\n/\nrate(reconcile_duration_seconds_count[$__rate_interval])",
```

Comment:
Shouldn't we keep both `reconcile_duration_seconds_sum` and `controller_runtime_reconcile_time_seconds_sum`?

Reply:
hmmm,  I would say they both convey the same information so it is unnecessary. Plus since we have more control over our custom metrics, it is open for extension in the future as opposed to the built-in metric

Reply:
hmm makes sense. But please change the value of the module name label to actually reflect the module name not the manifest name.